### DISCUSSION TITLE
Problem in completion status

### DISCUSSION THREAD

**Discussion by jaipal-reddy-p** [github/india#866] [General]
I have completed the azure services but the status is incomplete so please make it updated 
![Screenshot (2)](https://user-images.githubusercontent.com/96718212/180630470-24ffbaaf-29fa-4570-b46b-053bdd9aeb04.png)
![Screenshot (3)](https://user-images.githubusercontent.com/96718212/180630473-08ec9575-5703-4311-9d69-128c731157cd.png)
@futurereadyskills #125

**Comment by ayeshaumraoshaik** [comment]
Same problem to me 
How to get complete status....I tried a lot..but not getting it

**Comment by ayeshaumraoshaik** [comment]
Can u help me..please...
